x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/components/ordinal_legend.js

```diff
@@ -40,28 +38,20 @@ export class DynamicLegendRow extends React.Component {
     this._isMounted = true;
     this._loadParams();
   }
-
-  _formatValue(value) {
```

Comment:
This is still needed. You do not want to pass empty string to formatter. The formatter may not handle that in consistent ways. For example, maybe the formatter rounds numbers, how does that handle empty string?